main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap.md

```diff
@@ -177,52 +176,28 @@ export const makeNatAmountShape = (brand, min) =>
 
 Utility function to create a shape for amounts of the specified fungible brand. If a minimum value is provided, ensures the amount is greater than or equal to it.
 
-## Start Function
-Now we define the main entrypoint of the contract, the `start` function, with the usual arguments, `zcf`, `privateAge`, and `baggage`:
```

Comment:
I think we should keep explaining that the module exports a `start` function.

It's merely a convention/convenience that we define a more abstract `contract` and pass it to `withOrchestration`.

Reply:
@turadg  Will wait for your comments before merge.

Reply:
i see. lgtm